modules/ROOT/pages/single-sign-on-with-saml.adoc

```diff
@@ -159,7 +159,7 @@ NOTE: The expected format for Keys and certificates is PEM (with or without the
 . In the tenant_portal folder of each existing tenant: `<BUNDLE_HOME>/setup/platform_conf/current/tenants/<TENANT_ID>/tenant_portal`, +
 edit the file *keycloak-saml.xml* to setup Bonita webapp as a Service provider working with your IdP.
  ** The entityID is the Service Provider given to your bonita installation. You can change it if you want but you need to provide it to your IdP.
```

Comment:
your bonita installation.  => your Bonita installation.

Reply:
if you want but you need to provide it to your IdP. => if you want, but you need to provide it to your IdP.

Reply:
Done